### ISSUE TITLE
Config file 

### ISSUE THREAD

**Issue by mbkizil** [Victarry/Adaptation-SCR#1]
Hi,

There is no config directory or config file for Hydra. It produces the following error:

hydra.errors.MissingConfigException: Primary config directory not found.
Check that the config directory 'Adaptation-SCR/ZSSGAN/config' exists and readable

**Comment by hou-f** [comment]
Hello, did you resolve this problem?

**Comment by Victarry** [comment]
Hi, @hou-f , can you create a yaml file in `Adaptation-SCR/ZSSGAN/config/config.yaml`? And set the arguments in https://github.com/Victarry/Adaptation-SCR/blob/main/ZSSGAN/options/train_options.py. Feel free to ask if you still have issues.

**Comment by hou-f** [comment]
> Hi, [@hou-f](https://github.com/hou-f) , can you create a yaml file in `Adaptation-SCR/ZSSGAN/config/config.yaml`? And set the arguments in https://github.com/Victarry/Adaptation-SCR/blob/main/ZSSGAN/options/train_options.py. Feel free to ask if you still have issues.

Thanks for your reply and this awesome work! I will try it later!

**Comment by hou-f** [comment]
python train.py --config-path=config --config-name=config
D:\anaconda3\envs\ds\lib\site-packages\torch\utils\cpp_extension.py:2059: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation.
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
  warnings.warn(
D:\Desktop\code\Adaptation-SCR-main\ZSSGAN\train.py:100: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  @hydra.main(config_path="config", config_name="config")
D:\anaconda3\envs\ds\lib\site-packages\hydra\_internal\hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
  ret = run_job(
Initializing networks...
direction loss type cosine
relative loss type cosine
spectral loss type nmse
using adaptive spectral reg
  0%|                                                                                                                                    | 0/301 [00:00<?, ?it/s]D
:\Desktop\code\Adaptation-SCR-main\ZSSGAN\op\conv2d_gradfix.py:88: UserWarning: conv2d_gradfix not supported on PyTorch 2.6.0+cu124. Falling back to torch.nn.functional.conv2d().
  warnings.warn(
Clip loss: 1.040:   0%|▎                                                                                                     | 1/301 [03:53<19:28:40, 233.73s/it]

你好，请问训练时间为什么这么长呢？我使用的显示是rtx4060ti，下面是我的配置：

version: 1
# Training configuration
task: photo2sketch  # MISSING field must be specified
experiment: null
seed: 3
comment: ""
exp_path: ""

# Checkpoints
frozen_gen_ckpt: "../weights/stylegan2-ffhq-config-f.pt"
train_gen_ckpt: "../weights/stylegan2-ffhq-config-f.pt"
output_dir: "../Output"

# Loss weights and configurations
lambda_direction: 1.0
direction_loss_type: "cosine"

# Additional loss weights (unused in paper)
lambda_patch: 0.0
lambda_global: 0.0
lambda_texture: 0.0
lambda_manifold: 0.0
lambda_relative: 0.0
lambda_diversity: 0.0
lambda_contrastive: 0.0
lambda_ppl: 0.0
lambda_pplc: 0.0
lambda_kl: 0.0

# Spectral loss configuration
lambda_spectral: 1.0
input_space: "w+"
feature_space: "clip"
spectral_loss_type: "nmse"
spectral_reg_type: "trace"
spectral_weight_type: "adaptive"
power_iterations: 3
lerp_weight: 0
adaptive_weight: true
reg_frequency: 1

# Additional loss configurations
relative_loss_type: "cosine"

# Save and output intervals
save_interval: 50
output_interval: 50

# Class configuration
source_class: "photo"  # MISSING field must be specified
target_class: "sketch"  # MISSING field must be specified
phase: "default"

# Sampling and layer configurations
sample_truncation: 0.7
auto_layer_iters: 1
auto_layer_k: 18
auto_layer_batch: 8
clip_models: 
  - "ViT-B/32"
clip_model_weights:
  - 1.0

# Output configurations
num_grid_outputs: 1
crop_for_cars: false

# Image style configurations
style_img_dir: null
img2img_batch: 4
target_img_list: null

# Training parameters
iter: 301
batch: 4
n_sample: 2
size: 1024
mixing: 0.0
ckpt: null
lr: 0.002
channel_multiplier: 2
exp_name: null  # MISSING field must be specified
